### DISCUSSION TITLE
With less than 5k/10k traffic-what numbers can we realistically expect?

### DISCUSSION THREAD

**Discussion by syncronox** [merchantprotocol/theadslab#39] [Open Discussion Forum]
Just curious.

**Comment by jonathonbyrdziak** [comment]
It doesn't matter what your traffic numbers are, you'll always receive the same percentages across every industry.

Lower traffic websites typically complain about not getting enough leads, so we really don't have patience in dealing with those complaints. If you have low leads it's because you have a traffic problem. If you have less than 10k visitors per month, then you have a traffic problem.
